src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5763,11 +5777,14 @@ namespace HeatBalanceSurfaceManager {
             ZoneWinHeatLossRep = 0.0;
             ZoneWinHeatLossRepEnergy = 0.0;
 
-            CalcHeatBalanceInsideSurf2(dataZoneTempPredictorCorrector, DataSurfaces::AllHTSurfaceList,
-                                       DataSurfaces::AllIZSurfaceList,
-                                       DataSurfaces::AllHTNonWindowSurfaceList,
-                                       DataSurfaces::AllHTWindowSurfaceList);
-
+            if (AllCTF) {
+                CalcHeatBalanceInsideSurf2CTFOnly(1, NumOfZones, DataSurfaces::AllIZSurfaceList);
+            } else {
+                CalcHeatBalanceInsideSurf2(dataZoneTempPredictorCorrector, DataSurfaces::AllHTSurfaceList,
+                                           DataSurfaces::AllIZSurfaceList,
+                                           DataSurfaces::AllHTNonWindowSurfaceList,
+                                           DataSurfaces::AllHTWindowSurfaceList);
```

Comment:
`CalcHeatBalanceInsideSurf2CTFOnly` is called if everything is CTF *and* only when called for all zones. When called from radiant HVAC equipment to resimulate a single zone, `CalcHeatBalanceInsideSurf2` must be used instead, because the resimulated "zone" includes adjacent interzone surfaces.